src/DIRAC/Resources/Catalog/RucioFileCatalogClient.py

```diff
@@ -12,24 +12,30 @@
 
 import DIRAC
 from DIRAC import S_OK, S_ERROR, gLogger
+from DIRAC.Core.Security import Locations
 from DIRAC.Resources.Catalog.Utilities import checkCatalogArguments
 from DIRAC.Core.Utilities.List import breakListIntoChunks
-from DIRAC.Core.Security.ProxyInfo import getProxyInfo
+from DIRAC.Core.Security.ProxyInfo import getProxyInfo, getVOfromProxyGroup
 from DIRAC.Resources.Catalog.FileCatalogClientBase import FileCatalogClientBase
 
 from rucio.client import Client
 from rucio.common.exception import DataIdentifierNotFound, FileReplicaAlreadyExists, \
     CannotAuthenticate, MissingClientParameter
 from rucio.common.utils import chunks, extract_scope
 sLog = gLogger.getSubLogger(__name__)
+useDiracScopeAlg = False
 
 
 def get_scope(lfn, scopes=None):
-  """Helper function that extract the scope from the LFN
+  """Helper function that extracts the scope from the LFN
   """
-  if scopes is None:
-    scopes = []
-  scope, _ = extract_scope(did=lfn, scopes=scopes)
+  #  default Dirac scope algorithm:
+  if useDiracScopeAlg:
+    scope = lfn.split('/')[1]
+  else:
+    if scopes is None:
+      scopes = []
+    scope, _ = extract_scope(did=lfn, scopes=scopes)
```

Comment:
This method has a hidden access to the Rucio config file:
https://github.com/rucio/rucio/blob/master/lib/rucio/common/utils.py#L749
If you don't have `rucio.cfg` it will fail.

Reply:
OK, I'm assuming the this one will never be called if a Dirac scope algorithm is used. This is for config file collaboration only.